core/storage/suggestion/gae_models.py

```diff
@@ -129,7 +133,24 @@
 COMMUNITY_CONTRIBUTION_STATS_MODEL_ID: Final = 'community_contribution_stats'
 
 # Number of models to fetch for contributor admin dashboard stats.
-NUM_MODELS_PER_FETCH: Final = 100
+NUM_MODELS_PER_FETCH: Final = 500
+
+
+class SortChoice():
```

Comment:
Look at how other enums are done in the codebase, e.g. classes subclassing enum.Enum. Maybe you could use the same pattern.

Also the class name should be plural.

Reply:
Changed